src/T2DMap.cpp

```diff
@@ -4652,48 +4661,46 @@ void T2DMap::slot_setCustomLine2()
     if (!room) {
         return;
     }
-    if (exit == "NW") {
+    if (exit == QLatin1String("nw")) {
```

Comment:
:beetle: Unfortunately this code has subsequently proved to be buggy because `exit` is the *text* used on the clicked button in the form - and even the en_US case is broken because that used capital letters and not the lower case that this PR switches everything ***else*** over to...